letsencrypt-apache/letsencrypt_apache/tests/apache-conf-files/passing/ipv6-1143b.conf

```diff
@@ -0,0 +1,18 @@
+<VirtualHost *:443 [::]:443>
+DocumentRoot /tmp
+ServerName example.com
+ServerAlias www.example.com
+CustomLog ${APACHE_LOG_DIR}/example.log combined
+<Directory "/tmp">
+  AllowOverride All
+</Directory>
+
+  SSLEngine on
+
+  SSLHonorCipherOrder On
+  SSLProtocol all -SSLv2 -SSLv3
+  SSLCipherSuite "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH +aRSA RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS"
+
+   SSLCertificateFile /etc/ssl/certs/ssl-cert-snakeoil.pem
+   SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
```

Comment:
Yeah, it was, Travis doesn't have mod_headers, so it kept failing ( https://travis-ci.org/letsencrypt/letsencrypt/jobs/117270310). Also mod_headers is outside of the scope of this PR (and test).